src/SourceTerms/Deleptonization/CPU_Src_Deleptonization.cpp

```diff
@@ -282,12 +282,6 @@ static void Src_Deleptonization( real fluid[], const real B[],
       fluid[ENGY] = Hydro_ConEint2Etot( fluid[DENS], fluid[MOMX], fluid[MOMY], fluid[MOMZ], Eint_Update, Emag );
 
 
-//    update temperature initial guess
-#     ifdef TEMP_IG
-      fluid[TEMP_IG] = Out2[1];
-#     endif
-
-
       // final check
```

Comment:
Move `//` to the beginning of this line.

Reply:
I corrected this line.